mne_connectivity/spectral/tests/test_spectral.py

```diff
@@ -467,12 +467,19 @@ def test_spectral_connectivity(method, mode):
     assert out_lens[0] == 10
 
 
+_coh_marks = []
 _gc_marks = []
 if platform.system() == "Darwin" and platform.processor() == "arm":
+    _coh_marks.extend([
+        pytest.mark.filterwarnings("ignore:invalid value encountered in sqrt:")
```

Comment:
Can this be removed now that there is a `raise LinAlgError` in there? Or does it come from another code path?

Reply:
Yeah, the filters for the cohy methods should now be redundant. Have removed and will wait to see if macOS tests pass to be sure.